### ISSUE TITLE
Incorrect behaviour on engine translations in dummy app

### ISSUE THREAD

**Issue by adrigzr** [ember-intl/ember-intl#1151]
When adding `ember-intl` to a standalone Engine (engine within its own repository), translation engine files are no generated. Adding this Engine to a real App generates translation engine files correctly.

### Environment

- Ember Version: 3.1.6.0
- Ember CLI Version: 3.16.0
- Ember Intl Version: 4.2.3
- Browser(s): Chrome 79
- Node Version: 10.15.0

### Standalone engine

Clone and build the following repository:

```
$ git clone https://github.com/BBVAEngineering/intl-engine && cd intl-engine
$ npm install
$ ember build
$ ls dist/engines-dist/intl-engine/translations
```

**Expected**: Engine translation files are generated.

**Reality**: Engine translation files are **not** generated.

### App with engine

Clone and build the following repository:

```
$ git clone https://github.com/BBVAEngineering/intl-app && cd intl-app
$ npm install
$ ember build
$ ls dist/engines-dist/intl-engine/translations
```

Engine translation files are generated correctly.

**Comment by jasonmit** [comment]
Engines isn't something officially supported I'm but happy to review any changes to change this.

**Comment by jasonmit** [comment]
Fixed in 4.2.4
